calyx-py/calyx/builder.py

```diff
@@ -381,6 +381,330 @@ def fp_sop(
             ast.Stdlib.fixed_point_op(op_name, width, int_width, frac_width, True),
         )
 
+    def insert_comb_group(self, left, right, cell, groupname=None):
```

Comment:
Perhaps `binary_comb_group` to convey the fact that this only works for cells with two input ports, which must be named `left` and `right`?

Or even better, I would love it if this name followed the same convention as all the specific, per-operator functions below. So, like, `binary_use`, if we go with the convention that's currently used below.